cors.go

```diff
@@ -92,12 +104,18 @@ func Default() *Cors {
 func (cors *Cors) Handler(h http.Handler) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		if r.Method == "OPTIONS" {
+			if cors.options.Debug {
+				cors.options.log.Printf("Handler: Preflight request")
+			}
```

Comment:
I assume what you mean is why didn't I put the `log` call inside `handlePreflight` and `handleActualRequest`?

I did that deliberately because I wanted to know exactly while "entry point" was getting invoked.  You will notice that they have different prefixed (`Handler` vs `HandlerFunc` vs `ServeHTTP`).  Since it is debugging output (and would not appear normally), I didn't think it would be a big deal.  If you prefer that I do it the other way, I can remove these calls (since they are *only* here to identify the entry point).